docs/source/installation.rst

```diff
@@ -2,147 +2,123 @@ Installation
+        1. Setup Python Venv Virtual Environment.
 
-::
+        ``ivadomed`` requires Python >= 3.6 and <3.9 (If you are using `Compute Canada <https://www.computecanada.ca/>`_, you can load modules (e.g. python 3.9) as `mentioned here <https://intranet.neuro.polymtl.ca/computing-resources/compute-canada#modules>`_ and `also here <https://docs.computecanada.ca/wiki/Utiliser_des_modules/en#Loading_modules_automatically>`_ ). We recommend
+        working under a virtual environment, which could be set as follows:
 
-    git clone https://github.com/neuropoly/ivadomed.git
-    cd ivadomed
+        ::
 
-Step 2: Create new Conda Env called IvadoMedEnv (may take 5 to 15 minutes)
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+            python -m venv ivadomed_env
 
-::
+        2. Activate the new virtual environment (default named `ivadomed_env`)
 
-    conda env create --file environment.yml
+        ::
 
-Step 3 : Activate environment and use
-++++++++++++++++++++++++++++++++++++++
+            source ivadomed_env/bin/activate
 
-::
 
-    conda activate IvadoMedEnv
+        .. warning::
+           If the default Python version installed in your system does not fit the version requirements, you might need to specify a version of Python associated with your virtual environment:
 
-Step 4 : Install from source
-++++++++++++++++++++++++++++++++++++++
+           ::
 
-::
+             python -m venv ivadomed_env --python=python3.7
 
-    pip install -e .
+    .. tab:: Compute Canada HPC
 
-Approach 2: Venv
------------------
+        There are numerous constraints and limited package availabilities with ComputeCanada cluster environment.
 
-Step 1: Setup Python Virtual Environment.
-+++++++++++++++++++++++++++++++++++++++++
+        It is best to attempt ``venv`` based installations and follow up with ComputeCanada technicall support as MANY specially compiled packages (e.g. numpy) are exclusively available for Compute Canada HPC environment.
 
-``ivadomed`` requires Python >= 3.6 and <3.9 (If you are using `Compute Canada <https://www.computecanada.ca/>`_, you can load modules (e.g. python 3.9) as mentioned  `here <https://intranet.neuro.polymtl.ca/computing-resources/compute-canada#modules>`_ and `also here <https://docs.computecanada.ca/wiki/Utiliser_des_modules/en#Loading_modules_automatically>`_ ). We recommend
-working under a virtual environment, which could be set as follows:
 
-::
+Step 2: Install ``ivadomed``
+++++++++++++++++++++++++++++
 
-    virtualenv venv-ivadomed
-    source venv-ivadomed/bin/activate
 
+.. tabs::
 
-.. warning::
-   If the default Python version installed in your system does not fit the version requirements, you might need to specify a version of Python associated with your virtual environment:
+    .. tab:: Pypi Installation
 
-   ::
+        Install ``ivadomed`` and its requirements from
+        `Pypi <https://pypi.org/project/ivadomed/>`__:
```

Comment:
Nitpick: `Pypi` -> [`PyPI`](https://pypi.org/)

Reply:
Nice. TIL. Totally didn't know that before.